contracts/Datastore.sol

```diff
@@ -267,24 +315,39 @@ contract Datastore is AragonApp {
     }
 
     /**
-     * Sets IPFS as the storage provider for the datastore.
+     * @notice Change the encryption provider
+     * @param _encryptionProvider Encryption provider
+     */
+    function setEncryptionProvider(EncryptionProvider _encryptionProvider) public {
+        require(settings.encryptionProvider == EncryptionProvider.None);
+        settings.encryptionProvider = _encryptionProvider;
+        emit SettingsChanged(msg.sender);
```

Comment:
Wouldn't it be better to emit a more specific event, in order to know exactly which setting was changed?

Reply:
Yeah we are thinking of adding a param the will represent the type of change but we are still not sure if it's a good trade-off regarding the additional bytecode.